faf-commons-lobby/src/main/kotlin/com/faforever/commons/lobby/GameApi.kt

```diff
@@ -108,11 +115,12 @@ enum class VictoryCondition {
 // ***********************
 
 /**
+ * Info on current game or games.
  * GameInfo comes as single message or as a nested list
  * which makes all fields nullable. Thanks for nothing...
  */
 data class GameInfo(
-  val uid: Long?,
+  val uid: Int?,
```

Comment:
This looks dangerous. Is this intended?

Reply:
The id in the database is specified as an int and all other ids were ints so that is why I changed it. Could be changed back though